packages/components/src/components/icon/icons.js

```diff
@@ -530,13 +530,30 @@ import './underlying/ic-underlying-BOOM1000.svg';
 import './underlying/ic-underlying-BOOM500.svg';
 import './underlying/ic-underlying-CRASH1000.svg';
 import './underlying/ic-underlying-CRASH500.svg';
+import './underlying/ic-underlying-CRYBCHUSD.svg';
+import './underlying/ic-underlying-cryBNBUSD.svg';
```

Comment:
Can we use uppercase for `cry` to be consistent?
```suggestion
import './underlying/ic-underlying-CRYBNBUSD.svg';
```